File: src/nomanssave/dv.java
package nomanssave;

import java.awt.Container;
import java.awt.event.ActionEvent;
import java.awt.event.ActionListener;

class dv implements ActionListener {
   // $FF: synthetic field
   final du hp;
   // $FF: synthetic field
   private final gF hq;

   dv(du var1, gF var2) {
      this.hp = var1;
      this.hq = var2;
   }

   public void actionPerformed(ActionEvent var1) {
      if (this.hq != null) {
         cg.a((Container)du.d(this.hp), (gQ)this.hq);
         du.c(this.hp);
      }

   }
}

File: NMSSaveEditor/nomanssave/mixed/dV.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.IO;
using System.Linq;
using System.Text;
using System.Text.RegularExpressions;
using System.Windows.Forms;
using System.Globalization;

namespace NMSSaveEditor
{

public class dV : ActionListener {
   // $FF: synthetic field
   public dN ia;
   // $FF: synthetic field
   public Application bv;

   public dV(dN var1, Application var2) {
      this.ia = var1;
      this.bv = var2;
   }

   public void actionPerformed(ActionEvent var1) {
      gH var2 = (gH)dN.p(this.ia).SelectedItem;
      if (var2 != null) {
         eV var3 = this.bv.d("PlayerStateData.ShipUsesLegacyColours");
         if (var3 != null) {
            if (dN.f(this.ia).isSelected() ^ var3.ab(var2.getIndex())) {
               var3.a(var2.getIndex(), dN.f(this.ia).isSelected());
            }

         }
      }
   }
}

}
